src/include/stir/recon_buildblock/ProjMatrixByBin.h

```diff
@@ -155,7 +156,7 @@ class ProjMatrixByBin :
 
   
 protected:
-  shared_ptr<DataSymmetriesForBins> symmetries_ptr;
+  shared_ptr<DataSymmetriesForBins> symmetries_sptr;
```

Comment:
unfortunate but necessary change. Add it to release notes for developers (as breaks any derived class from ProjMatrixByBin)